.github/workflows/ci.yml

```diff
@@ -44,7 +44,8 @@ jobs:
       matrix:
         os: ["ubuntu-22.04"]
         # TODO: eclipse-velocitas/cli/issues/141
-        node_version: [lts/-1, lts/*, 20.4.0]
+        # node_version: [lts/-1, lts/*, 20.4.0]
+        node_version: [lts/-1, 20.4.0]
```

Comment:
please use lts/Iron and its/Hydrogen

Reply:
doesnt work because hydrogen is >20.4.0

Reply:
And what is exactly issue with >20.4.0 ??

Reply:
we are using a mock-fs which has a bug for node >20.4.0